### PR TITLE
Refactor/update icons

### PR DESCRIPTION
**Pull request by Samsunggora**
image
image (1)

### CODE REVIEW

**Comment by Pe5h4** [comment]
@Samsunggora Ok, this looks good, dont forget to update asab-webui submodule, when it will be ready/approved

**Comment by Pe5h4** [comment]
@Samsunggora Resolve conflicts pls

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Release Candidate
 
+### Refactoring
+
+- Changed icons from coreui.io icons to atlasicons. (INDIGO Sprint 230428, [!29](https://github.com/TeskaLabs/seacat-auth-webui/pull/29))
```
Comment (petrKavulok) [inline CHANGELOG.md]: Please update sprint number.

Comment (petrKavulok) [reply CHANGELOG.md]: Also, don't forget to version bump asab-webui sumbodule

**Review by petrKavulok — CHANGES_REQUESTED** [review]

**Pull request merged**